doc/design/volumes/nfsvolumestores.md

```diff
@@ -114,20 +137,28 @@ func VolumesList() {
 #### Functional
 
  1. Create a VCH with an NFS backed `VolumeStore`, create a volume on the `VolumeStore`, create 2 containers with the volume attached, touch a file from the first container, verify it exists on the 2nd.  Destroy the 2nd container, attempt to destroy the volume and expect a failure.  Poweroff the first container, reattempt destroy of the volume, it should fail.  Then destroy the container and destroy the volume. 
- 2. Create a VCH with a nonreachable NFS backed `VolumeStore`.  Creation of the volume should return an error.
+ 2. Create a VCH with a nonreachable NFS backed `VolumeStore`.  Check that vic-machine warns about the lack of creation of that volume store. Possibly check for the appropriate logging in the portlayer logs. 
+ 3. Create a VCH with an NFS backed `VolumeStore`, create a volume on said store. Attach this volume to two different cotainers. Touch a file in the volume with container 1. Confirm that this file shows up in container 2. remove the VCH and confirm that the volume store is removed as well. 
+ 4. Create a VCH with an NFS volume store. create a volume attach it to a container. touch a file to that container. Tear down the VCH. Create a new VCh with the same NFS volume store target. Check that the created volume is still there. Attach it to a container. Check that the touched file appears inside the volume. 
+ 5. Create a volume store with multiple NFS Volume Stores. Create volumes on each store. Mount each of those volumes to the same container. Touch files to them. attach all volumes to a second container. Make sure that all touched files are present (multiple volume version of test 3)
+ 6. Create a VCH with an nfs volume store. Create a volume attach it to a running container. Attempt to remove the volume. Expect to get a `Volume in Use` error. 
+ 7. Create a VCH with an nfs store tagged as the `default` store. Test all the functionalities of the anonymous volume functionality.(this will likely be multiple tests, there are already several tests for our vdmk based volume store to mirror from.)
```

Comment:
`Create a VCH with an nfs store tagged as the ``default store.`` Test all the functionalities of the anonymous volume functionality (this will likely be multiple tests, there are already several tests to mirror from for our vdmk based volume store).`